include/rmm/mr/device/fixed_size_memory_resource.hpp

```diff
@@ -209,12 +206,15 @@ class fixed_size_memory_resource
     return block_type{ptr};
   }
 
+  // Unnamed parameters are not supported by doxygen,
```

Comment:
Should we name this parameter and mark it as unused instead? Just wondering how this shows up in the docs. We don't want it to look like there's no parameters.

Reply:
I'm fairly certain unnamed but unused parameters will trigger a compiler warning. I can verify though.

Reply:
Just make sure the docs look right, that’s the priority imo.

Reply:
`[[ maybe_unused ]]` works.